packages/core/src/components/graphs/area.ts

```diff
@@ -0,0 +1,125 @@
+// Internal Imports
+import { Component } from "../component";
+import * as Configuration from "../../configuration";
+import { CartesianOrientations, Events } from "../../interfaces";
+
+// D3 Imports
+import { area } from "d3-shape";
+
+export class Area extends Component {
+	type = "area";
+
+	init() {
+
```

Comment:
extra space here
```suggestion
```

Reply:
wow just noticed this. I'll add with the stacked area PR